snmp/assets/dashboards/interface_performance.json

```
@@ -0,0 +1,521 @@
+{
+  "title": "Interface Performance",
+  "description": "Use this dashboard to view the performance of all the interfaces on your network devices. This dashboard can be filtered across a specific interfaces to view the specific performance of one single interface, or across a device by name to see the entire device's performance. (cloned)",
+  "widgets": [
+    {
+      "id": 0,
+      "layout": { "x": 21, "y": 7, "width": 56, "height": 19 },
+      "definition": {
+        "title": "Inbound Bandwidth (%) ",
+        "title_size": "16",
+        "title_align": "left",
+        "show_legend": false,
+        "legend_size": "0",
+        "type": "timeseries",
+        "requests": [
+          {
+            "q": "avg:snmp.ifBandwidthInUsage.rate{$snmp_host,$interface,$snmp_device} by {snmp_host,interface}",
+            "on_right_yaxis": false,
+            "style": {
+              "palette": "cool",
+              "line_type": "solid",
+              "line_width": "normal"
+            },
+            "display_type": "line"
+          }
+        ],
+        "yaxis": {
+          "scale": "linear",
+          "label": "",
+          "include_zero": true,
+          "min": "auto",
+          "max": "auto"
+        },
+        "markers": []
+      }
+    },
+    {
+      "id": 1,
+      "layout": { "x": 78, "y": 7, "width": 56, "height": 19 },
+      "definition": {
+        "title": "Outbound Bandwidth (%)",
+        "title_size": "16",
+        "title_align": "left",
+        "show_legend": false,
+        "legend_size": "0",
+        "type": "timeseries",
+        "requests": [
+          {
+            "q": "avg:snmp.ifBandwidthOutUsage.rate{$snmp_host,$interface,$snmp_device} by {interface,snmp_host}",
+            "on_right_yaxis": false,
+            "style": {
+              "palette": "cool",
+              "line_type": "solid",
+              "line_width": "normal"
+            },
+            "display_type": "line"
+          }
+        ],
+        "yaxis": {
+          "scale": "linear",
+          "label": "",
+          "include_zero": true,
+          "min": "auto",
+          "max": "auto"
+        },
+        "markers": []
+      }
+    },
+    {
+      "id": 2,
+      "layout": { "x": 21, "y": 0, "width": 113, "height": 6 },
+      "definition": {
+        "type": "note",
+        "content": "Utilization",
+        "background_color": "blue",
```

```suggestion
        "background_color": "vivid blue",
```

In the guidelines, main metric group headers are `vivid blue` and subcategory headers are `blue`.
https://datadoghq.dev/integrations-core/guidelines/dashboards/#metric-categories